engine/evd.ml

```diff
@@ -948,20 +948,18 @@ let mkLEvar sigma (evk, args) =
   let args = List.fold_right2 fold (evar_filtered_context evi) args SList.empty in
   mkEvar (evk, args)
 
+let is_relevance_irrelevant sigma r =
+  match UState.nf_relevance sigma.universes r with
+  | Irrelevant -> true
+  | Relevant -> false
+  | RelevanceVar _ -> false (* Should check whether the variable is registered *)
```

Comment:
no need for a comment here
we already have the FIXME comment in ustate qstate.repr